core_backend/app/db/db_models.py

```diff
@@ -316,71 +321,258 @@ class ContentDB(Base):
     SQL Alchemy data model for content
     """
 
-    __tablename__ = "content"
+    __tablename__ = "contents"
 
     content_id: Mapped[int] = mapped_column(Integer, primary_key=True, nullable=False)
+
+    def __repr__(self) -> str:
+        """Pretty Print"""
+        return f"<Content #{self.content_id}>"
+
+
+class LanguageDB(Base):
+    """
+    SQL Alchemy data model for language
+    """
+
+    __tablename__ = "languages"
+
+    language_id: Mapped[int] = mapped_column(Integer, primary_key=True, nullable=False)
+
+    language_name: Mapped[str] = mapped_column(String, nullable=False)
+    is_default: Mapped[bool] = mapped_column(Boolean, nullable=False, default=False)
+    created_datetime_utc: Mapped[datetime] = mapped_column(DateTime, nullable=False)
+    updated_datetime_utc: Mapped[datetime] = mapped_column(DateTime, nullable=False)
```

Comment:
I learned that the advantage of mapped_column is being able to infer column types from type hints, reducing boilerplate code! (https://docs.sqlalchemy.org/en/20/orm/declarative_tables.html#using-annotated-declarative-table-type-annotated-forms-for-mapped-column )

So we don't have to import sqlalchemy specific types and can write something like
```suggestion
    language_id: Mapped[int] = mapped_column(primary_key=True)
    language_name: Mapped[str] = mapped_column()
    is_default: Mapped[bool] = mapped_column(default=False)
    created_datetime_utc: Mapped[datetime] = mapped_column()
    updated_datetime_utc: Mapped[datetime] = mapped_column()
```

if you don't use Optional[] then its by default `nullable=False` apparently..

Reply:
I see all of our other code has some redundancies.. we can clean this up in another PR.

Reply:
Yes! I think so as well.  We can do that in the refactor PR.